addons/kotsadm/template/base/install.sh

```diff
@@ -1,13 +1,11 @@
 #!/usr/bin/env bash
 
-function kotsadm_pre_init() {
-    validate_object_storage
```

Comment:
This function will always fail here because the Rook/Minio addons create the necessary ENV variables afterward.